docs/literate/src/files/first_steps/create_first_setup.jl

```diff
@@ -0,0 +1,258 @@
+#src # Create first setup
+
+# In this part of the introductory guide, we will create a first Trixi.jl setup as an extension of
+# [`elixir_advection_basic.jl`](https://github.com/trixi-framework/Trixi.jl/blob/main/examples/tree_2d_dgsem/elixir_advection_basic.jl).
+# Since Trixi.jl has a common basic structure for the setups, you can create your own by extending
+# and modifying the following example.
+
+# Let's consider the linear advection equation for a state ``u = u(x, y, t)`` on the two-dimensional spatial domain
+# ``[-1, 1] \times [-1, 1]`` with a source term
+# ```math
+# \frac{\partial}{\partial t}u + \frac{\partial}{\partial x} (0.2 u) - \frac{\partial}{\partial y} (0.7 u) = - 2 e^{-t}
+# \sin\bigl(2 \pi (x - t) \bigr) \sin\bigl(2 \pi (y - t) \bigr)
+# ```
+# and with an initial condition
+# ```math
+# u(x, y, 0) = \sin\bigl(\pi x \bigr) \sin\bigl(\pi y \bigr).
+# ```
+
+# The first step is to create and open a file with the .jl extension. You can do this with your
+# favorite text editor. In this file you will create your setup.
+
+# To be able to use functionalities of Trixi.jl, you always need to load Trixi.jl itself
+# and the [OrdinaryDiffEq.jl](https://github.com/SciML/OrdinaryDiffEq.jl) package.
+
+using Trixi
+using OrdinaryDiffEq
+
+# The next thing to do is to choose an equation that is suitable for your problem. To see all the
+# currently implemented equations, take a look at
+# [`src/equations`](https://github.com/trixi-framework/Trixi.jl/tree/main/src/equations).
+# If you are interested in adding a new physics model that has not yet been implemented in
+# Trixi.jl, take a look at the tutorials
+# [Adding a new scalar conservation law](@ref adding_new_scalar_equations) or
+# [Adding a non-conservative equation](@ref adding_nonconservative_equation).
+
+# The linear scalar advection equation in two spatial dimensions
+# ```math
+# \frac{\partial}{\partial t}u + \frac{\partial}{\partial x} (a_1 u) + \frac{\partial}{\partial y} (a_2 u) = 0
+# ```
+# is already implemented in Trixi.jl as
+# [`LinearScalarAdvectionEquation2D`](@ref), for which we need to define a two-dimensional parameter
+# `advection_velocity` describing the parameters ``a_1`` and ``a_2``. Appropriate for our problem is `(0.2, -0.7)`.
+
+advection_velocity = (0.2, -0.7)
+equations = LinearScalarAdvectionEquation2D(advection_velocity)
+
+# To solve our problem numerically using Trixi.jl, we have to discretize the spatial
+# domain, for which we set up a mesh. One of the most used meshes in Trixi.jl is the
+# [`TreeMesh`](@ref). The spatial domain used is ``[-1, 1] \times [-1, 1]``. We set an initial number
+# of elements in the mesh using `initial_refinement_level`, which describes the initial number of
+# hierarchical refinements. The variable `n_cells_max` is used to limit the number of elements in the mesh,
```

Comment:
We should mention that the number of leaf cells = actual elements is computed as 2^initial_refinement_level

Reply:
```suggestion
# of elements in the mesh using `initial_refinement_level`, which describes the initial number of
# hierarchical refinements. In this simple case, the total number of elements is `2^initial_refinement_level`
# throughout the simulation. The variable `n_cells_max` is used to limit the number of elements in the mesh,
```